tests/licensedcode/test_detect.py

```diff
@@ -1311,3 +1311,16 @@ def test_detection_return_correct_mit_not_apache_using_full_index(self):
         matches = idx.match(location=query_location)
         results = [m.rule.license_expression for m in matches]
         assert results == expected
+
+    def test_detection_returns_correct_no_gpl3_false_positive(self):
+
+        idx = cache.get_index()
+        expected = []  
+
+        query_location = self.get_test_loc('false_positive/false-positive-gpl3.txt')
+        matches = idx.match(location=query_location)
+
+        results = [m.rule.license_expression for m in matches]
```

Comment:
There is no need to do this as matches is a empty list. You could just do something like `assert not matches`